api/src/plugins/csrf.ts

```diff
@@ -23,7 +23,7 @@ const csrf: FastifyPluginCallback = (fastify, _options, done) => {
     // tokens since we know we can provide this one
     getToken: req => req.headers[CSRF_HEADER] as string,
     cookieOpts: { signed: false, sameSite: 'strict' },
-    logLevel: 'debug'
+    logLevel: 'trace'
```

Comment:
@ojeytonwilliams do you know what this prop is for? Can we remove this thing?

Reply:
What do you want to remove, the logging or the option? If we don't specify the options, it will go back to logging the CSRF errors as `warn`. If you want to supress the logging entirely, it needs setting to `silent`.

Reply:
Ah - I guess this is what I have been seeing all this while. I guess we will leave it at `debug` but I think this is the message I have been seeing all along

https://github.com/fastify/csrf-protection/blob/41e378801efa00af2321766e8ec715e8e50d1a09/index.js#L123

Reply:
Yeah, it is. I introduced the `logLevel` option so that we could control how that error appears in our logs.

Once we're sure that it's not legit users being bounced for bad reasons, we can make them `silent` or `trace` so they're not filling our log service with spam.